web-app/src/lib/db/extension/agent.ts

```diff
@@ -59,7 +59,17 @@ export function getCredits(agent: AgentWithFixedPricing): number {
   if (!agent.pricing.fixedPricing) {
     throw new Error("Agent must have FixedPricing");
   }
-  return Number(agent.pricing.fixedPricing.amounts[0].amount);
+  const unit = agent.pricing.fixedPricing.amounts[0].unit;
+  const amount = Number(agent.pricing.fixedPricing.amounts[0].amount);
+
+  switch (unit) {
+    case "usdm":
+      return amount;
```

Comment:
yes that is not how usdm is displayed, its the policy id, but I am switching it over to the calculateCredits functiona t the moment